app/index/components/navigationBox/navigationBox.js

```diff
@@ -0,0 +1,9 @@
+const navigationBox = {
+	name: 'box',
+	body: `<div class="flex-item">
+    <a href="{{item}}.html"><img class="index__navigation-box--images" src="./app/index/images/{{item}}.png">
```

Comment:
The `href` values here end up being wrong, since they aren't all lowercase, and sometimes have spaces.

Reply:
check (this is just so i know what i have done)